bugsnag-spring/src/test/java/com/bugsnag/SpringMvcTest.java

```diff
@@ -202,6 +202,7 @@ public void beforeNotify(Report report) {
         } finally {
             // Remove the callback via reflection so that subsequent tests do not use it
             Field callbacksField = Configuration.class.getDeclaredField("callbacks");
+            callbacksField.setAccessible(true);
```

Comment:
Why is this change necessary?

Reply:
Because callbacks were changed from public to private. This can be reverted if we're going to back out the accessors.